test/test_transforms_tensor.py

```diff
@@ -24,194 +24,207 @@
 NEAREST, BILINEAR, BICUBIC = InterpolationMode.NEAREST, InterpolationMode.BILINEAR, InterpolationMode.BICUBIC
 
 
-class Tester(unittest.TestCase):
-
-    def setUp(self):
-        self.device = "cpu"
+def _test_transform_vs_scripted(transform, s_transform, tensor, msg=None):
```

Comment:
The diff makes the changes look very complicated but I really didn't modify the logic of any of the `_test....` helpers here